libs/transactions/src/lib/directives/transaction-additional-details.directive.ts

```diff
@@ -1,7 +1,15 @@
 import { Directive, TemplateRef } from "@angular/core";
+import { TransactionItem } from "@backbase/data-ang/transactions";
 
+export type AdditionalDetailsContext = {
+  $implicit: TransactionItem
```

Comment:
we could either combine a key of `$implicit` with `let-transaction` in the template

or `transaction: TransactionItem` in combination with `let-transaction="transaction"`

Reply:
I would suggest to narrow down the example to the additions field in a transaction item (if exists) and avoid passing entire object in

Reply:
@cheshirsky  Sure I can do that. So it's not a common scenario that, when implementing a journey, we want to add one of the existing properties to the view?

Reply:
Not sure if it makes much sense to type the context in that case, since the additions are very loosely typed as `[key: string]: string;`